apps/public-docsite/src/components/IconGrid/IconGrid.tsx

```diff
@@ -1,40 +1,98 @@
 import * as React from 'react';
-import { Icon, SearchBox } from '@fluentui/react';
+import { FontIcon, ISearchBoxProps, SearchBox, Spinner } from '@fluentui/react';
 import * as stylesImport from './IconGrid.module.scss';
 const styles: any = stylesImport;
 
-export interface IIconGridProps {
+export interface IFontIconGridProps {
   /**
-   * An array of icons.
+   * An array of font-based icon names.
    */
   icons: { name: string }[];
 
   /**
-   * If we should render using `Icon` from Fabric
+   * Type of icons:
+   * - `core-font` is Fabric Core font icons, rendered in an `<i>` tag with classes
+   * - `react-font` is Fluent UI React font icons, rendered using `<FontIcon>`
+   * - `react-svg` is SVG icon components, rendered directly
```

Comment:
react-svg appears to be in the next interface

Reply:
This one is a bit weird since IFontIconGridProps is never used directly, only in IIconGridProps (where `iconType` is the discriminant), and I wanted the full comment to show up on the final IIconGridProps type. At least in my editor, it was using the comment from the first type for the combined version, rather than merging the comments.